src/libmscore/draw/qpainterprovider.h

```diff
@@ -61,45 +61,18 @@ class QPainterProvider : public IPaintProvider
     void save() override;
     void restore() override;
 
-    void setWorldTransform(const QTransform& matrix, bool combine = false) override;
-    const QTransform& worldTransform() const override;
-
-    void setTransform(const QTransform& transform, bool combine = false) override;
+    void setTransform(const QTransform& matrix) override;
```

Comment:
Can you rename matrix to transform, please?

Reply:
renamed
(although transform and matrix are essentially the same :))